controllers/operatorpolicy_controller.go

```diff
@@ -26,18 +28,26 @@ const (
 	OperatorControllerName string = "operator-policy-controller"
 )
 
+var (
+	subscriptiongvk  = schema.GroupVersionKind{Group: "operators.coreos.com", Version: "v1", Kind: "OperatorGroup"}
+	operatorgroupgvk = schema.GroupVersionKind{Group: "operators.coreos.com", Version: "v1", Kind: "OperatorGroup"}
```

Comment:
```suggestion
	subscriptionGVK  = schema.GroupVersionKind{Group: "operators.coreos.com", Version: "v1", Kind: "OperatorGroup"}
	operatorGroupGVK = schema.GroupVersionKind{Group: "operators.coreos.com", Version: "v1", Kind: "OperatorGroup"}
```

Reply:
@zyjjay After pulling changes I noticed that these are duplicates as well, subscription should be `schema.GroupVersionKind{Group: "operators.coreos.com", Version: "v1alpha1", Kind: "Subscription"}`